src/AutoRest.CSharp/AutoRest/Plugins/Configuration.cs

```diff
@@ -26,9 +28,11 @@ public Configuration(string outputFolder, string? ns, string? name, string[] sha
             CredentialTypes = credentialTypes;
             CredentialScopes = credentialScopes;
             CredentialHeaderName = credentialHeaderName;
+            OperationGroupToResourceType = operationGroupToResourceType?.ValueKind == JsonValueKind.Null ? new Dictionary<string, string>() : JsonSerializer.Deserialize<Dictionary<string, string>>(operationGroupToResourceType.ToString());
+            OperationGroupToResource = operationGroupToResource?.ValueKind == JsonValueKind.Null ? new Dictionary<string, string>() : JsonSerializer.Deserialize<Dictionary<string, string>>(operationGroupToResource.ToString());
+            ResourceRename = resourceRename?.ValueKind == JsonValueKind.Null ? new Dictionary<string, string>() : JsonSerializer.Deserialize<Dictionary<string, string>>(resourceRename.ToString());
```

Comment:
nit: should we add a helper

Reply:
yeah, this will go into a future pr https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/5726